src/main/java/command/commandName/Print.java

```diff
@@ -7,13 +7,14 @@
     @Override
     public void execute(String command) {
 
-        if (command.equals("print")) {
-
-            StorageTask.getAllTask().entrySet().stream()
+        switch (command) {
+            case "print" -> StorageTask.getAllTask().entrySet().stream()
                     .filter(a -> !a.getValue().isStatus())
-                    .forEach(a -> CommandImpl.out(a.getKey(), a.getValue()));
-        } else {
-            StorageTask.getAllTask().forEach(CommandImpl::out);
+                    .forEach(a -> super.out(a.getKey(), a.getValue()));
```

Comment:
нет необходимости обращаться к `super`. Т.к. метод `protected` - он доступен наследникам `CommandImpl`.
```suggestion
                    .forEach(a -> out(a.getKey(), a.getValue()));
```